numpy/distutils/fcompiler/gnu.py

```diff
@@ -287,7 +289,7 @@ def version_match(self, version_string):
                     self.executables[key].append('-mno-cygwin')
         return v
 
-    possible_executables = ['gfortran', 'f95']
+    possible_executables = ['gfortran', 'f95', '/usr/bin/gfortran']
```

Comment:
```suggestion
    possible_executables = ['gfortran', 'f95']
```
This was originally to help f2py find my Fortran compiler, but that doesn't seem to be an issue anymore.  Should I drop this change?